"GL_INVALID_OPERATION error generated. No active compute shader."

I installed the latest master branch of render pipeline and the latest development version of Panda3D for windows. It seems that OpenGL shaders will not compile on my Windows system running the latest NVIDIA drivers. I get this error:

```
fatal error C9999: *** exception during compilation ***
:display:gsg:glgsg(error): An error occurred while linking GLSL shader
:display:gsg:glgsg(error): GL_INVALID_OPERATION error generated. No active compute shader.
```

See the example below:
```
C:\Panda3D-1.10.0-x64\RenderPipeline-master\RenderPipeline-master\samples\01-Material-Demo>C:\Panda3D-1.10.0-x64\python\python.exe main.py
[>] CORE                      Using panda3d-supplied core module
[>] RenderPipeline            Using Python 3.6 with architecture win_amd64
[>] RenderPipeline            Using Panda3D 1.10.0 built on Sep  9 2018 15:09:21
[>] RenderPipeline            Using git commit 47496068d33de66ea479168b78501305bf63fe49
[>] MountManager              Auto-Detected base path to /c/Panda3D-1.10.0-x64/RenderPipeline-master/RenderPipeline-master
[>] RenderPipeline            Mount manager was not mounted, mounting now ...
[>] MountManager              Setting up virtual filesystem
[>] MountManager              Mounting auto-detected config dir: /c/Panda3D-1.10.0-x64/RenderPipeline-master/RenderPipeline-master/config/
[>] MountManager              Mounting ramdisk as /$$rptemp
[>] RenderPipeline            No settings loaded, loading from default location
Known pipe types:
  wglGraphicsPipe
(all display modules loaded.)
[>] RenderPipeline            Driver Version = 4.3.0 NVIDIA 399.24
[>] RenderPipeline            Driver Vendor = NVIDIA Corporation
[>] RenderPipeline            Driver Renderer = GeForce GTX 1050 Ti with Max-Q Design/PCIe/SSE2
[>] RenderPipeline            Render resolution is 800 x 600
[>] LightManager              Tile size = 24 x 16 , Num tiles = 34 x 38
[>] GPUCommandQueue           Allocating command buffer of size 32768
[>] GroupedInputBlock         Native UBO support = False
[!] RPLoader                  Loading '/$$rp/data/default_cubemap/cubemap.txo' took 178.62 ms
[>] PluginManager             Loading plugin settings
[>] PluginManager             Creating plugin instances ..
[>] Debugger                  Creating debugger
[>] plugin:scattering         Loading scattering method for 'eric_bruneton'
[>] StageManager              Setup stages ..
[>] StageManager              Preparing stages ..
[>] AOStage                   Blur quality is MEDIUM
[>] StageManager              Awaiting future pipe Exposure
[>] UpdatePreviousPipesStage  Creating previous pipes stage ..
[>] StageManager              Writing shader config
[>] plugin:pssm               Initializing pssm ..
[>] ScatteringMethodEricBruneton Precomputing ...
:display:gsg:glgsg(error): An error occurred while compiling GLSL compute shader /$$rp/rpplugins/scattering/shader/eric_bruneton/transmittance.compute.glsl:
(1093) : fatal error C9999: *** exception during compilation ***
:display:gsg:glgsg(error): An error occurred while linking GLSL shader
:display:gsg:glgsg(error): GL_INVALID_OPERATION error generated. No active compute shader.
:display:gsg:glgsg(error): An error occurred while compiling GLSL compute shader /$$rp/rpplugins/scattering/shader/eric_bruneton/delta_e.compute.glsl:
(1091) : fatal error C9999: *** exception during compilation ***
:display:gsg:glgsg(error): An error occurred while linking GLSL shader
:display:gsg:glgsg(error): GL_INVALID_OPERATION error generated. No active compute shader.
:display:gsg:glgsg(error): An error occurred while compiling GLSL compute shader /$$rp/rpplugins/scattering/shader/eric_bruneton/delta_sm_sr.compute.glsl:
(1092) : fatal error C9999: *** exception during compilation ***
:display:gsg:glgsg(error): An error occurred while linking GLSL shader
:display:gsg:glgsg(error): GL_INVALID_OPERATION error generated. No active compute shader.
:display:gsg:glgsg(error): An error occurred while compiling GLSL compute shader /$$rp/rpplugins/scattering/shader/eric_bruneton/copy_irradiance.compute.glsl:
(1095) : fatal error C9999: *** exception during compilation ***
:display:gsg:glgsg(error): An error occurred while linking GLSL shader
:display:gsg:glgsg(error): GL_INVALID_OPERATION error generated. No active compute shader.
:display:gsg:glgsg(error): An error occurred while compiling GLSL compute shader /$$rp/rpplugins/scattering/shader/eric_bruneton/copy_inscatter.compute.glsl:
(1094) : fatal error C9999: *** exception during compilation ***
:display:gsg:glgsg(error): An error occurred while linking GLSL shader
:display:gsg:glgsg(error): GL_INVALID_OPERATION error generated. No active compute shader.
:display:gsg:glgsg(error): An error occurred while compiling GLSL compute shader /$$rp/rpplugins/scattering/shader/eric_bruneton/delta_j.compute.glsl:
(1097) : fatal error C9999: *** exception during compilation ***
:display:gsg:glgsg(error): An error occurred while linking GLSL shader
:display:gsg:glgsg(error): GL_INVALID_OPERATION error generated. No active compute shader.
:display:gsg:glgsg(error): An error occurred while compiling GLSL compute shader /$$rp/rpplugins/scattering/shader/eric_bruneton/irradiance_n.compute.glsl:
(1096) : fatal error C9999: *** exception during compilation ***
:display:gsg:glgsg(error): An error occurred while linking GLSL shader
:display:gsg:glgsg(error): GL_INVALID_OPERATION error generated. No active compute shader.
:display:gsg:glgsg(error): An error occurred while compiling GLSL compute shader /$$rp/rpplugins/scattering/shader/eric_bruneton/delta_sr.compute.glsl:
(1093) : fatal error C9999: *** exception during compilation ***
:display:gsg:glgsg(error): An error occurred while linking GLSL shader
:display:gsg:glgsg(error): GL_INVALID_OPERATION error generated. No active compute shader.
:display:gsg:glgsg(error): An error occurred while compiling GLSL compute shader /$$rp/rpplugins/scattering/shader/eric_bruneton/add_delta_e.compute.glsl:
(1097) : fatal error C9999: *** exception during compilation ***
:display:gsg:glgsg(error): An error occurred while linking GLSL shader
:display:gsg:glgsg(error): GL_INVALID_OPERATION error generated. No active compute shader.
:display:gsg:glgsg(error): An error occurred while compiling GLSL compute shader /$$rp/rpplugins/scattering/shader/eric_bruneton/add_delta_sr.compute.glsl:
(1097) : fatal error C9999: *** exception during compilation ***
:display:gsg:glgsg(error): An error occurred while linking GLSL shader
[>] RenderPipeline            Finished initialization in 4.346 s, first frame: 3
[>] RenderPipeline            Took 0.783 s until first frame
```

--- a/rpcore/render_pipeline.py
+++ b/rpcore/render_pipeline.py
@@ -587,7 +587,7 @@ class RenderPipeline(RPObject):
         # Provide driver vendor as a define
         vendor = self._showbase.win.gsg.get_driver_vendor().lower()
         defines["IS_NVIDIA"] = "nvidia" in vendor
-        defines["IS_AMD"] = "ati" in vendor
+        defines["IS_AMD"] = vendor.startswith("ati")
         defines["IS_INTEL"] = "intel" in vendor
 
         defines["REFERENCE_MODE"] = self.settings["pipeline.reference_mode"]

